MAINTAINERS.md

```diff
@@ -17,3 +17,17 @@ Song Gao ([@Yisaer](https://github.com/Yisaer)) | [gaosong@pingcap.com](mailto
 Calvin Weng ([@dcalvin](https://github.com/dcalvin)) | [wenghao@pingcap.com](mailto:wenghao@pingcap.com) | [PingCAP](https://www.pingcap.com/)
 Ben Ye ([@yeya24](https://github.com/yeya24)) | [yb532204897@gmail.com](mailto:yb532204897@gmail.com) | Individual
 Hengliang Tan ([@Gallardot](https://github.com/Gallardot)) | [tttick@gmail.com](mailto:tttick@gmail.com) | [Xpeng Motors](https://www.xiaopeng.com/)
+
+
+The Committers of Chaos Mesh, along with their emails and focus area, are listed below:
+
+| Name                         | Email                    | Org     |
+|------------------------------|--------------------------|---------|
+| Glen Yang ([[G1eny0ung](https://github.com/g1eny0ung))  | [yangyue@pingcap.com](mailto:yangyue@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Yihao Fu ([fewdan](https://github.com/fewdan))            | [fuyihao@pingcap.com](mailto:fuyihao@pingcap.com)      | [PingCAP](https://www.pingcap.com/) |
+| Jiaxing Wu ([Colstuwjx](https://github.com/Colstuwjx) )       | [wjx_colstu@hotmail.com](mailto:wjx_colstu@hotmail.com)             |   [Kingnet](https://www.kingnet.com/)      |
```

Comment:
Should be "([Colstuwjx](https://github.com/Colstuwjx))" ?   It looks like there is an extra space.